README.adoc

```diff
@@ -127,6 +127,19 @@ NOTE: Debug output from `vagrant-service-manager` is prepended with
 the following string:
 `DEBUG command: [ service-manager: <command name / log message> ]`
 
+=== Configurations
+
+- HTTP Proxy Settings +
+User can able to start ADB in proxy environment by specifying following configurations
+through Vagrantfile as:
+
++
+----------------------------------------------------------------------------
+config.servicemanager.http_proxy = <Proxy URL>
+config.servicemanager.http_proxy_user = <Proxy user name>
+config.servicemanager.http_proxy_password = <Proxy user password>
+----------------------------------------------------------------------------
+
```

Comment:
I think on vagrant reload, it will not work since we are not invoking our hooks on it. Probably we need to add it. When add it, it means it would be a separate feature since I don't remember we have taken care of this (vagrant reload) use case.